client/src/partials/journal/modals/columnConfig.modal.js

```diff
@@ -0,0 +1,33 @@
+angular.module('bhima.controllers')
+.controller('ColumnsConfigModalController', ColumnsConfigModalController);
+
+ColumnsConfigModalController.$inject = [ '$uibModalInstance', 'columnList', 'defaultColumns'];
+
+/**
+ * @module cash/modals/CashTransferModalController
```

Comment:
This isn't the cash modal.  Could you please update your comments when you copy from another module?